onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -55,6 +55,9 @@ BasicBackend::BasicBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
       config["MYRIAD_HW_INJECT_STAGES"] = CONFIG_VALUE(NO);
       config["MYRIAD_COPY_OPTIMIZATION"] = CONFIG_VALUE(NO);
     }
+
+    //to check preprocessing for yolov3
+    config["MYRIAD_CHECK_PREPROCESSING_INSIDE_MODEL"] = CONFIG_VALUE(NO);
```

Comment:
just to confirm, this is a good default value for all models (not just yolov3) ?

Reply:
I just checked it with Bakalin, Andrew(openvino team) - recommendation for user to use MYRIAD_CHECK_PREPOCESSING_INSIDE_MODEL = NO configuration option in each case when preprocessing steps are external from model (so it's not implemented as first layers in the model)

This is true for all models which are processed through UEP without Model Optimizer usage, as I understand

Reply:
thanks for the confirmation!